src/models/ticket.rs

```diff
@@ -24,8 +24,8 @@ impl std::fmt::Display for Ticket {
     fn fmt(&self, f: &mut Formatter<'_>) -> Result<(), Error> {
         writeln!(
             f,
-            "Ticket:\n\tId:{:?}\n\tTitle:{}\n\tDescription:{}\n\tStatus:{:?}",
-            self.id, self.title, self.description, self.status
+            "Ticket:\n\tId:{:?}\n\tTitle:{}\n\tDescription:{}\n\tStatus:{:?}\n\tComments:{:?}",
```

Comment:
It's coming using an array-like syntax because you are relying on the `Debug` formatter, `{:?}`.
Arrays do not implement the `Display` formatter, hence you need to deal with them directly.
You can have multiple `write!` and `writeln!` statement in `fmt`, hence you could use a for loop to iterator over the comments and print them out with the right indentation.

Reply:
See https://doc.rust-lang.org/std/fmt/#formatting-traits

Reply:
I have updated this. I tried something similar last night but didn't get multiple writes in the fmt to work. I think it is because I didn't use the ? operator at the end of each writeln!. Thank you for your tip and patience! 👌👍